src/cargo_build.rs

```diff
@@ -586,3 +587,109 @@ impl CargoBuilder {
         self
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::str::FromStr;
+
+    #[rustversion::stable]
+    const IS_RUST_STABLE_VERSION: bool = true;
+    #[rustversion::any(beta, nightly)]
+    const IS_RUST_STABLE_VERSION: bool = false;
```

Comment:
It's worth adding a TODO that these should get removed once rust 1.68.0 is released.